src/System.Diagnostics.TextWriterTraceListener/src/FxCopBaseline.cs

```diff
@@ -0,0 +1,3 @@
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Microsoft.Reliability", "CA2002:DoNotLockOnObjectsWithWeakIdentity", Scope = "member", Target = "System.Diagnostics.DelimitedListTraceListener.#set_Delimiter(System.String)")]
```

Comment:
What is the purpose of the lock in this case?  It seems to be pointless in the current implementation... the right fix may be to just remove the lock rather than suppressing the warning.